core/controllers/admin.py

```diff
@@ -141,14 +176,68 @@ def post(self):
                     'new_config_property_values')
                 logging.info('[ADMIN] %s saved config property values: %s' %
                              (self.user_id, new_config_property_values))
+                config_properties = (
+                    config_domain.Registry.get_config_property_schemas())
                 for (name, value) in new_config_property_values.iteritems():
                     config_services.set_property(self.user_id, name, value)
+                    # Below checks are for maintaining the sync between roles
+                    # in old and new authorization system.
+                    # NOTE: This block of code is going to be removed once the
```

Comment:
I think convention is to use `TODO(your_name): blah` rather than `NOTE`. Putting your name after a TODO isn't a commitment to address it, it's just the name of the person who inserted the todo.